frontend/src/host_orchestrator/api/v1/messages.go

```diff
@@ -167,3 +167,44 @@ type StartCVDRequest struct {
 	// Start from the relevant snaphost if not empty.
 	SnapshotID string `json:"snapshot_id,omitempty"`
 }
+
+type DisplayParams struct {
+	Width         int `json:"width"`
+	Height        int `json:"height"`
+	Dpi           int `json:"dpi,omitempty"`
+	RefreshRateHz int `json:"refresh_rate_hz,omitempty"`
+}
+
+type DisplayAddRequest struct {
+	Displays []*DisplayParams `json:"displays"`
+}
+
+type DisplayAddResponse = EmptyResponse
+
+type DisplayMode struct {
+	Windowed []int `json:"windowed"`
+}
+
+type Display struct {
+	Dpi           []int       `json:"dpi"`
+	Mode          DisplayMode `json:"mode"`
+	RefreshRateHz int         `json:"refresh-rate"`
+}
+
+type DisplayListResponse struct {
+	Displays map[int]Display `json:displays`
+}
+
+type DisplayRemoveRequest struct {
+	Displays []int `json:"displays"`
+}
+
+type DisplayRemoveResponse = EmptyResponse
+
+type DisplayScreenshotRequest struct {
+	DisplayNumber int `json:"display_number,omitempty"`
+}
+
+type DisplayScreenshotResponse struct {
```

Comment:
This should include the image format (mime type?).

Reply:
Added.